cmd/createtree/main.go

```diff
@@ -141,16 +142,28 @@ func newPK(opts *createOpts) (*any.Any, error) {
 	switch opts.privateKeyType {
 	case "PEMKeyFile":
 		if opts.pemKeyPath == "" {
-			return nil, errors.New("empty PEM path")
```

Comment:
Please keep the checks - they are useful for tool users (and slightly more readable than the resulting server-side error message they would get).